src/indicator/renderer/forceIndex.js

```diff
@@ -0,0 +1,72 @@
+import d3 from 'd3';
+import annotationLine from '../../annotation/line';
+import _multi from '../../series/multi';
+import {noop} from '../../util/fn';
+import seriesLine from '../../series/line';
+
+export default function() {
+
+    var xScale = d3.time.scale(),
+        yScale = d3.scale.linear(),
+        multiSeries = _multi(),
+        decorate = noop;
+
+    var annotations = annotationLine();
+
+    var forceLine = seriesLine()
+        .yValue(function (d, i) {
```

Comment:
I wouldn't expect a space between the `function` keyword and the opening bracket `(`